ui/app/components/mount-backend-form.js

```diff
@@ -112,18 +112,21 @@ export default Component.extend({
           throw err;
         }
       }
-
+      // TODO CMB: this flash message seems to be an over-generalization, so added the 'if...path !==' check, but want to revisit
+      // TODO should we check instead if config capabilities.capabilites.includes('deny')
       if (!capabilities.get('canUpdate')) {
```

Comment:
we have access to `capabilities: {capabilities: 'deny'}` wondering if that would be better here as `canUpdate` feels a little misleading

Reply:
I don't think `deny` === `!canUpdate`, since they could have read capabilities but not update.